<launch>
    <include file="$(find lane_detection)/launch/lane_detection.launch"></include>

    <!-- lane detection camera -->
    <node name="usb_cam_1" pkg="usb_cam" type="usb_cam_node" output="screen" >
        <param name="video_device" value="/dev/video0" />
        <param name="image_width" value="864" />
        <param name="image_height" value="480" />
        <param name="pixel_format" value="yuyv" />
        <param name="camera_frame_id" value="usb_cam" />
        <param name="io_method" value="mmap"/>
    </node>
</launch>